pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / EmuConfig.GS.FramerateNTSC.ToFloat() : (fps * 100) / EmuConfig.GS.FrameratePAL.ToFloat(); 
```

Comment:
So what happens with `Region_NTSC_PROGRESSIVE`? (it won't work - will assume PAL rate while it should target NTSC rate).